### ISSUE TITLE
Fix illegal invocation using FormData proxy

### ISSUE THREAD

**Issue by scrhartley** [bigskysoftware/htmx#2995]
Currently the `htmx:configRequest` event exposes `detail.parameters` as a proxied FormData.
When trying to use it for debugging with:
```js
element.addEventListener('htmx:configRequest', function(event) {
    console.log(new URLSearchParams(event.detail.parameters).toString());
});
```
I get:
> Uncaught TypeError: Illegal invocation

To work around this, it currently needs to be converted to entries:
```js
element.addEventListener('htmx:configRequest', function(event) {
    console.log(new URLSearchParams(event.detail.parameters.entries()).toString());
});
```

With a normal FormData object, it can be passed directly:
```js
let data = new FormData();
data.set('foo', 'bar');
console.log(new URLSearchParams(data).toString()); // Outputs foo=bar
```

I have a fix and will create a PR if that's OK: https://github.com/scrhartley/htmx/commit/bdd17de17a851052a9ba54eefb0aa991686196f0

**Comment by Telroshan** [comment]
Hey, bugfix PRs welcome @scrhartley! Didn't think of this usecase when writing that proxy, good fix!

**Comment by scrhartley** [comment]
Fixed by #2997
